src/components/NotificationsDrawer/DrawerPanel.tsx

```diff
@@ -0,0 +1,515 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { PopoverPosition } from '@patternfly/react-core/dist/dynamic/components/Popover';
+import { Badge } from '@patternfly/react-core/dist/dynamic/components/Badge';
+import { useAtom, useAtomValue, useSetAtom } from 'jotai';
+import {
+  ChromeWsEventTypes,
+  ChromeWsPayload,
+} from '@redhat-cloud-services/types';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import {
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core/dist/dynamic/layouts/Flex';
+import {
+  Dropdown,
+  DropdownGroup,
+  DropdownItem,
+  DropdownList,
+} from '@patternfly/react-core/dist/dynamic/components/Dropdown';
+import {
+  MenuToggle,
+  MenuToggleElement,
+} from '@patternfly/react-core/dist/dynamic/components/MenuToggle';
+import { Divider } from '@patternfly/react-core/dist/dynamic/components/Divider';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+} from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import {
+  NotificationDrawerBody,
+  NotificationDrawerHeader,
+  NotificationDrawerList,
+} from '@patternfly/react-core/dist/dynamic/components/NotificationDrawer';
+// import { DrawerContent } from '@patternfly/react-core/dist/dynamic/components/Drawer';
+import {
+  NotificationData,
+  NotificationsPayload,
+  addNotificationAtom,
+  notificationDrawerDataAtom,
+  notificationDrawerFilterAtom,
+  notificationDrawerSelectedAtom,
+  updateNotificationReadAtom,
+  updateNotificationSelectedAtom,
+  updateNotificationsSelectedAtom,
+} from '../../state/atoms/notificationDrawerAtom';
+import { Text } from '@patternfly/react-core/dist/dynamic/components/Text';
+import { Title } from '@patternfly/react-core/dist/dynamic/components/Title';
+import FilterIcon from '@patternfly/react-icons/dist/dynamic/icons/filter-icon';
+import BellSlashIcon from '@patternfly/react-icons/dist/dynamic/icons/bell-slash-icon';
+import EllipsisVIcon from '@patternfly/react-icons/dist/dynamic/icons/ellipsis-v-icon';
+import orderBy from 'lodash/orderBy';
+import { Link, useNavigate } from 'react-router-dom';
+import NotificationItem from './NotificationItem';
+import BulkSelect from '@redhat-cloud-services/frontend-components/BulkSelect';
+import axios from 'axios';
+import { Access } from '@redhat-cloud-services/rbac-client';
+import {
+  Stack,
+  StackItem,
+} from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { getSevenDaysAgo } from '../UtcDate';
+
+interface Bundle {
+  id: string;
+  name: string;
+  displayName: string;
+  children: Bundle[];
+}
+
+interface FilterConfigItem {
+  title: string;
+  value: string;
+}
+
+export type DrawerPanelProps = {
+  panelRef: React.Ref<unknown>;
+  isOrgAdmin: boolean;
+  toggleDrawer: () => void;
+  getUserPermissions: (
+    applicationName?: string,
+    disableCache?: boolean
+  ) => Promise<Access[]>;
+};
+
+const EmptyNotifications = ({
```

Comment:
Can we break this into a new file? Having multiple components in one file makes it harder to debug and read.